lang/cs/ReefDotNet.sln

```diff
@@ -54,106 +53,140 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{79E7F89A-1DFB-45E1-8D43-D71A954AEB98}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
We are getting rid of CBT build. Moving to Apache, we should be able to support both x64 and  any CPU, right?